performance/gcd_bench.cpp

```diff
@@ -0,0 +1,215 @@
+
```

Comment:
Do we need the copyright notice here?

Reply:
Yes, just seen that in the failed inspection report, I'll wait for the other CI tests to cycle before making a few cosmetic changes like this.